### PR TITLE
Fix stretched home video on Edge

### PR DESCRIPTION
**Pull request by taniaprocha**

### CODE REVIEW

File: src/components/Background/index.css [hunk 1]
```diff
@@ -55,11 +55,14 @@
...
-  left: -50%;
+  top: 0px;
+  top: 0px;
 
-  width: 200%;
-  height: 200%;
+  width: auto;
+  height: auto;
+
+  min-width: 100%;
```
Comment (jfranciscosousa) [inline src/components/Background/index.css]: Don't forget to check on IE 11

Comment (taniaprocha) [reply src/components/Background/index.css]: The video shows up right on IE, just for a second and 💣 (also on svhealth.io)

Comment (filipebfernandes) [reply src/components/Background/index.css]: Dude, aren't you supposed to be on vacation? 😆

Comment (jfranciscosousa) [reply src/components/Background/index.css]: Just took a couple minutes off of my "off time" 😂

Comment (taniaprocha) [reply src/components/Background/index.css]: Well, I create the PR #92, but IE continue to 💣. But we talk about this next year 😁

**Pull request merged**